app/main.py

```diff
@@ -1,8 +1,50 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int, clean_mark: int,
+                 brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
+                 distance_from_city_center: int,
+                 clean_power: int,
+                 average_rating: int,
+                 count_of_ratings: int) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list[Car]) -> float:
+        income = 0
+        for car in cars:
+            income += self.wash_single_car(car)
+            if car.clean_mark < self.clean_power:
+                car.clean_mark = self.clean_power
+        return round(income, 1)
+
+    def calculate_washing_price(self, car: Car) -> float:
+        output = (
+            car.comfort_class
+            * (self.clean_power - car.clean_mark)
+            * self.average_rating
+            / self.distance_from_city_center
```

Comment:
The `calculate_washing_price` method divides by `self.distance_from_city_center`. If `distance_from_city_center` is zero, this will result in a division by zero error. Consider adding a check to prevent this scenario.